### DISCUSSION TITLE
Custom command in popup

### DISCUSSION THREAD

**Discussion by wybryk-ananasa** [NeogitOrg/neogit#1037] [Q&A] [answered]
How can I add custom command to popup like commit or pull?

**Comment by CKolkey** [comment, marked as answer]
Ehh, short answer is you can't. Could be cool to open up an interface for it though.

You can always fork the project and mess with these files: https://github.com/NeogitOrg/neogit/blob/master/lua/neogit/popups/commit/init.lua

Should be quite straightforward to add stuff there.
